### ISSUE TITLE
US 01.05.01

### ISSUE THREAD

**Issue by ArsalanFiroozkoohi** [CMPUT301F24butter/butter-project#38]
As an entrant I want another chance to be chosen from the waiting list if a selected user declines an invitation to sign up
## Acceptance Criteria
- Notifications Page must be implemented before starting. (US 01.04.01, US 01.04.02, US 01.04.03)
- This is handled in the backend, but if a Chosen Entrant declines an invitation to sign up, then a new random Entrant will be drawn from the Waiting List by the Organizer.
- So in short, this will happen by default. The entrant does not have to do anything.
